src/resources/language/_language-fr.yml

```diff
@@ -51,3 +51,22 @@ crossref-eq-prefix: "Équation"
 crossref-lof-title: "Liste des Figures"
 crossref-lot-title: "Liste des Tables"
 crossref-lol-title: "Liste des Listings"
+
+listing-page-order-by: "Trier par"
+listing-page-order-by-default: "Présélectionné"
+listing-page-order-by-date-asc: "Le plus ancien"
+listing-page-order-by-date-desc: "Le plus récent"
+listing-page-order-by-number-desc: "Descendant"
+listing-page-order-by-number-asc: "Ascendant"
```

Comment:
Did not found where this is shown in the english version. Wanted to see translation in context, but it seems good.

Reply:
This would only be shown for pretty advanced cases like when a user defines some metadata field that is numeric, then includes that metadata in the list of sortable fields (so it just represents a numeric value sorted ascending or descending)...